### DISCUSSION TITLE
rabbitmq 3.8.2 not sending messages to DLX after message TTL

### DISCUSSION THREAD

**Discussion by obbyK** [rabbitmq/rabbitmq-server#8534] [Questions] [answered]
Is this a known bug of this version? I tested this on my local machine with version 3.11.16 and the message was dead lettered after message ttl (on queue) but in version 3.8.2 the message just stays there even after the ttl has been surpaased.

**Comment by michaelklishin** [comment, marked as answer]
RabbitMQ 3.8 is [has reached end of life](https://rabbitmq.com/versions.html) 11 months ago, so I'm afraid we will not investigate. All [release notes](https://rabbitmq.com/changelog.html) are public and you can search the repo for PRs that mention DLX.

**Reply by obbyK** [reply]
Will try and do some more searching. Thanks for the reply.

**Reply by obbyK** [reply]
found out the quorum message ttl support started from version 3.10
